systems/rolls.md

```diff
@@ -1,9 +1,22 @@
-## Advantage:
+## Advantage
 
-Advantage is a an additional `+1d4` that is added to your roll. These are given due to advantageous circumstances that improve your odds of succeeding your task. Advantage may stack indefintely, and is the opposite of [disadvantage](rolls#Disadvantage).
+Advantage is a an additional `+1d4` that is added to your roll. These are given due to advantageous circumstances that improve your odds of succeeding your task. Advantage may stack indefintely, and is the opposite of [disadvantage](#Disadvantage).
 
 Advantage and disadvantage cancel out before rolling. Ie, if you have 1x advantage and 3x disadvantage, you would add `-2d4` to your roll.
 
-## Disadvantage:
+## Disadvantage
+
+Disadvantage is a `-1d4` that is added to your roll. See [advantage](#Advantage) for information on stacking.
+
+## Tests
+
+Tests are a roll that your character makes against a foe to perform some kind of action. This is done by comparing one of your [attributes](stats.md#Attributes) against an atribute of your foes.
+
+Your score is formed by rolling `2d6` plus your characters atribute score. Your foes score is the average value of their attribute, or `7` plus their attribute score. If your score equals or exceeds your foes score, you succeed the test.

```

Comment:
This could be worded as `Your foes score is '7' plus their attribute score (the average value of their attribute)` to remove confusion as to whether there are two options here.